docs/tools-cli.md

```diff
@@ -0,0 +1,120 @@
+---
+title: CLI Helpers
+---
+
+If you end up working with large data sets such as the latest dump from Singular, we recommend using the CLI tools to generate this json data for you, in case you experience a large CPU data spike when testing out your frontend. You can read the original CLI documentation below or in the [rmrk-tools github repo](https://github.com/rmrk-team/rmrk-tools).
+
+### How to install CLI helpers
+
+You can use this package as a CLI tool: `yarn add rmrk-tools@latest --dev`
+
+Now you can use any of the helper commands in your bash or npm scripts by prepending `rmrk-tools-` to the helper name:
+
+```
+"scripts": {
+    "fetch": "rmrk-tools-fetch",
+    "consolidate": "rmrk-tools-consolidate",
+    "seed": "rmrk-tools-seed",
+}
+```
+
+Or in bash:
+
+```
+#! /usr/bin/env node
+import shell from "shelljs";
+
+shell.exec(
+  'rmrk-tools-fetch --ws wss://node.rmrk.app --prefixes=0x726d726b,0x524d524b --append=dumps/latest.json',
+);
+```
+
+### Additional notes
+
+- If you get a build error when you try using the cli helper tools, you need to make sure you install all of your dependencies (for me, it required that I install the dependencies `@polkadot/api` and `arg`).
+
+> "arg is for local dev cli tools." --Ilia
+
+- For those working with latest dump data: if you run the cli:`fetch` method, and you are still trying to connect to the 127.0.0.1:9944 default endpoint, you need to run fetch with the optional parameter `--ws wss://node.rmrk.app` or whatever you decided on your local node endpoint.
+
+## Fetch
+
+Grabs all system.remark extrinsics in a block range and logs an array of them all, keyed by block.
+
+> "This is a generalized fetcher that can fetch any remarks, not just RMRK nfts." --Bruno
+
+```
+// example
+yarn fetch --ws wss://node.rmrk.app --prefixes=0x726d726b,0x524d524b --append=dumps/latest.json
+```
+
+The return data will look like this:
+
+```
+[
+  {
+    block: 8,
+    call: [
+      {
+        call: "system.remark",
+        value: "0x13371337",
+      },
+      {
+        call: "balances.transfer",
+        value:
+          "5CK8D1sKNwF473wbuBP6NuhQfPaWUetNsWUNAAzVwTfxqjfr,10000000000000000",
+      },
+    ],
+  },
+  {
+    block: 20,
+    call: [
+      {
+        call: "system.remark",
+        value: "0x13371338",
+      },
+    ],
+  },
+];
+```
+
+Optional parameters for `fetch`:
```

Comment:
You're missing escapes here:

\- --ws URL

vs

\- `--ws URL`


Also not sure this is really necessary - we'll keep the README of the tools up to date. By copying it here, you risk going out of date in two places.